db/migrate/20221027100246_add_user_ref_to_like.rb

```diff
@@ -0,0 +1,8 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :author, foreign_key: { to_table: :users }
+    add_reference :likes, :post, foreign_key: {to_table: :posts}
+    add_reference :comments, :author, foreign_key: { to_table: :users }
+    add_reference :comments, :post, foreign_key: {to_table: :posts}
+  end
+end
```

Comment:
The original migration file for the `likes.rb` should look something like this
![image](https://user-images.githubusercontent.com/81932478/198260607-7d31cbc7-5b55-4197-9a16-320c2b84d429.png)
- [ ] So since youre adding the referential integrity seperately, it sould look like this
![image](https://user-images.githubusercontent.com/81932478/198260946-16b63d6a-7a7d-4b67-b02d-2d1f8f1d885c.png)

Reply:
well resolved! ✔️